abs.yaml

```
@@ -0,0 +1,3 @@
+aggregate_branch: libxml_2.13.1
+channels:
+  - https://staging.continuum.io/prefect/fs/libxslt-feedstock/pr4/9bb69e4
```

```suggestion
  - https://staging.continuum.io/prefect/fs/libxslt-feedstock/pr4/9bb69e4
  - https://staging.continuum.io/prefect/fs/libxml2-feedstock/pr17/1c2e862
```
Could you rebuild against the latest fix of libxml2?

---

I'm pretty confident this is the correct version? https://github.com/AnacondaRecipes/libxml2-feedstock/pull/17/commits/1c2e8626c9d8dc0d05913d1d9a9b4ae8eccf4577

---

Yes, it's correct.